### ISSUE TITLE
crates.io: 0.6.1 using 0.6.0 version of gluon_format

### ISSUE THREAD

**Issue by jgrowl** [gluon-lang/gluon#367]
Hi! I think there is an issue with the 0.6.1 version of gluon published on crates.io.

I tried bumping gluon to 0.6.1 on my project and got the following error:

```
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: failed to select a version for `gluon_base` (required by `gluon_format`):
all possible versions conflict with previously selected versions of `gluon_base`
  version 0.6.1 in use by gluon_base v0.6.1
  possible versions to select: 0.6.0
```
I then noticed that gluon_format is at `^0.6.0` on https://crates.io/crates/gluon/0.6.1

It looks like you intended to keep all gluon sub-projects at the same version number so I assume this is an error.

I didn't see any references to 0.6.0 in the code on a brief look through so I'm not sure if maybe something weird just happened on crates.io.

Sorry if I'm just doing something dumb and thanks for your efforts on this project!

**Comment by Marwes** [comment]
Thanks for reporting! I did see that the `gluon` crate accidentally had a dependency on `gluon_format` when doing the last release but even though I removed it I accidentally uploaded it before that.

I will upload 0.6.2 tonight (together with a fix for another bug which occured in 0.6.1).

**Comment by jgrowl** [comment]
Closing this issue as it looks to be fixed in 0.6.2. Thanks again! 👍
